api4/plugin.go

```diff
@@ -66,7 +66,11 @@ func uploadPlugin(c *Context, w http.ResponseWriter, r *http.Request) {
 	}
 	defer file.Close()
 
-	manifest, unpackErr := c.App.InstallPlugin(file, false)
+	force := false
+	if len(m.Value["force"]) > 0 && m.Value["force"][0] != "" {
+		force = true
```

Comment:
I think we should explicitly check against `true` here, otherwise if I upload with `force=false`, the behaviour seems incorrect.